src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -128,6 +127,8 @@ public DatabaseRecord(string databaseName)
 
         public HashSet<string> UnusedDatabaseIds = new HashSet<string>();
 
+        public long? DatabaseVersion;
```

Comment:
I would go here directly with dictionary telling what we support, instead of using a number.
Would be much simpler..

Reply:
done